src/oaklib/converters/owl_to_fhir_codesystem.py

```diff
@@ -0,0 +1,233 @@
+"""Convert OWL to FHIR CodeSystem JSON
+
+Resources
+- https://incatools.github.io/ontology-access-kit/interfaces/obograph.html
+- https://github.com/biopragmatics/bioontologies
+- https://bioontologies.readthedocs.io/en/latest/usage.html
+- https://build.fhir.org/codesystem.html
+
+TODO's
+  * Move this into GitHub issue (and link here) so I can get input on some of these more difficult parts / parts I don't yet understand
+  1. Write converter MVP
```

Comment:
you could also look at pyrophen as a model of how to do this

Reply:
Just FYI, I checked Pyrophen. It is simple enough to understand. My intuition is still leaning towards Obographs being faster and easier to do.